modules/passkey/README.md

```diff
@@ -2,9 +2,29 @@
+
+The below sections give a high-level overview of the contracts present in this package.
+
+### [SafeWebAuthnSignerProxy](./contracts/SafeWebAuthnSignerProxy.sol)
+
+A proxy contract is uniquely deployed for each Passkey signer. The signer information i.e, Public key co-ordinates, Verifier address, Singleton address are immutable. All calls to the signer are forwarded to the `SafeWebAuthnSignerSingleton` contract.
+
+Use of `SafeWebAuthnSignerProxy` provides gas savings compared to the whole contract deployment for each signer creation. The `SafeWebAuthnSignerProxy` contract is a proxy contract that forwards calls to the `SafeWebAuthnSignerSingleton` contract which is a singleton contract. Both `SafeWebAuthnSignerProxy` and `SafeWebAuthnSignerSingleton` use no storage slots to avoid storage access violations defined in ERC-4337. The details on gas savings can be found in [this PR](https://github.com/safe-global/safe-modules/pull/370). This proxy contract appends public key co-ordinates, and verifier information to the calldata before forwarding call to the singleton contract.
+
+### [SafeWebAuthnSignerSingleton](./contracts/SafeWebAuthnSignerSingleton.sol)
+
+`SafeWebAuthnSignerSingleton` implements ERC-1271 interface to support signature verification which enables signature data to be forwarded from a Safe to the `WebAuthn` library. The `SafeWebAuthnSignerSingleton` contract is a singleton contract that forwards calls to the `WebAuthn` library. This contract expects public key co-ordinates, and verifier address to be appended by the caller inspired from [ERC-2771](https://eips.ethereum.org/EIPS/eip-2771).
+
+### [SafeWebAuthnSignerFactory](./contracts/SafeWebAuthnSignerFactory.sol)
+
+The `SafeWebAuthnSignerFactory` contract is used to deploy the `SafeWebAuthnSignerProxy` contract with the public key co-ordinates and verifier information. The factory contract also supports signature verification for the public-key and signature information without deploying the signer contract which is used during the validation of `UserOp`. Using [ISafeSignerFactory](./contracts/interfaces/ISafeSignerFactory.sol) interface and this factory contract address, new signers can be deployed .
+
+### [WebAuthn](./contracts/WebAuthn.sol)
```

Comment:
The signature scheme documentation is notably missing

Reply:
https://github.com/safe-global/safe-modules/pull/394/commits/25815c35006a1db8fc546c3d9fd282c1b9a96567